arrows/qt/widgets/LoggerView.cxx

```diff
@@ -0,0 +1,71 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#include "LoggerView.h"
+
+#include "ui_LoggerView.h"
+
+namespace kv = kwiver::vital;
+
+namespace kwiver {
+namespace arrows {
+namespace qt {
+
+//-----------------------------------------------------------------------------
+class LoggerViewPrivate
+{
+public:
+
+  Ui::LoggerView UI;
```

Comment:
should this be `m_UI`?

Reply:
That's a good question .. m_UI is consistent with kwiver style but I think the way you have it is more Qt consistent. I'm ok with the Qt style in a Qt class since that's the general pattern we've been following.

Reply:
Per comments on the previous PR in this series, I'd say "yes". Qt vs. KWIVER style is (in my book, anyway) `m_camelCase` vs. `m_words_with_underscores`. (On that note, while I'm fine with `m_UI`, I've also seen and used `m_ui`.)